### PR TITLE
Update INL HPC directions

### PR DESCRIPTION
**Pull request by cticenhour**
@simopier This set of instructions works using the `moose-dev` module on Sawtooth. Apptainer instructions for HPC were recently uploaded in the MOOSE repository, but these are very WIP. For now we'll stick with a manual build until we have something better. 

Refs #26

### CODE REVIEW

File: doc/content/getting_started/hpc.md [hunk 1]
```diff
@@ -47,9 +47,54 @@ module load use.moose moose-dev
...
+<!-- End getting started re-use -->
+
+Once MOOSE is downloaded, several MOOSE dependencies need to be built: PETSc, libMesh, and WASP.
+These can be built easily using update-and-rebuild scripts within MOOSE. To speed up these build
+times, it is highly recommended to set the environment variable `MOOSE_JOBS`, which will speed up
+compilation time by using more than one core to build the dependency. One can set this variable to
+signal that the scripts use of 8 computer cores by doing the following:
+
+```bash
+export MOOSE_JOBS=8
```
Comment (simopier) [inline doc/content/getting_started/hpc.md:81]:
```suggestion
export MOOSE_JOBS=4 METHODS=opt 
```
Didn't @loganharbour send a message on slack about this needing to be 4 now? 
I'm also adding METHODS to build opt.

File: doc/content/getting_started/hpc.md [hunk 2]
```diff
@@ -84,9 +129,23 @@ not need to be kept up-to-date by the end user/developer. These are updated auto
...
 
-!include getting_started/installation.md start=To update your TMAP8 repository as a TMAP8 user end=Finally, TMAP8 can be re-compiled and re-tested. include-end=True
+!include getting_started/installation.md start=To update your TMAP8 repository as a TMAP8 user end=commands as necessary.
+
+If there were updates to MOOSE dependencies, the update-and-rebuild scripts should be re-run. It is
+only necessary to re-run those scripts for dependencies that were actually updated! It is usually
+helpful to check the most recent [MOOSE Newsletter](https://mooseframework.inl.gov/newsletter/index.html) or [Discussion Forum](https://github.com/idaholab/moose/discussions) for any update announcements from the MOOSE development team for the preceding month.
+
+```bash
+cd ~/projects/TMAP8/moose
```
Comment (simopier) [inline doc/content/getting_started/hpc.md:139]:
```suggestion
export MOOSE_JOBS=4 METHODS=opt 
cd ~/projects/TMAP8/moose
```

**Review by simopier — CHANGES_REQUESTED** [review]
Thank you! A couple suggestions.

File: doc/content/getting_started/hpc.md (hunk 1 above)
Comment (cticenhour) [reply doc/content/getting_started/hpc.md:81]: Broadcasted after I made this 😄

Comment (simopier) [reply doc/content/getting_started/hpc.md]: Hahaha I figured 😄 .

**Review by simopier — APPROVED** [review]
Thank you @cticenhour

**Pull request merged**